lib/widgets/mxc_image.dart

```diff
@@ -191,22 +191,19 @@ class _MxcImageState extends State<MxcImage> {
   Widget placeholder(BuildContext context) =>
       widget.placeholder?.call(context) ??
       const Center(
+        key: Key(placeholderKey),
```

Comment:
I don't understand why you use key here, because key is the same with all image is `placeholder`